bin/build

```diff
@@ -27,10 +27,12 @@ echo "cache = true" > ${layers_dir}/shim.toml
 "${target_dir}/bin/compile" "$(pwd)" "${cache_dir}" "${platform_dir}/env"
 
 # copy profile.d scripts into a layer so they will be sourced
-profile_dir="${layers_dir}/profile"
-mkdir -p "${profile_dir}/profile.d"
-cp .profile.d/* "${profile_dir}/profile.d/"
-echo "launch = true" > "${profile_dir}.toml"
+if [ -d .profiled.d ]; then
```

Comment:
`profiled.d` or `profile.d`?

Reply:
I don't think so, since the `cp` command is `cp .profile.d/* "${profile_dir}/profile.d/"`

Reply:
oops, I see. Fixing.